src/analysis_and_optimization/Optimize.ml

```diff
@@ -249,6 +249,19 @@ let inline_list f es =
   let es = List.map ~f:(function _, _, x -> x) dse_list in
   (d_list, s_list, es)
 
+let compute_suffix_and_name propto suffix fname =
+  let open Fun_kind in
+  match suffix with
+  | FnLpdf propto' when propto' && propto ->
+      ( FnLpdf true
+      , with_unnormalized_suffix fname |> Option.value ~default:fname )
```

Comment:
Why do we need the unnormalized function name here? When this is removed, test output changes but only for stanlib functions which makes no sense to me.

Reply:
In `create_function_inline_map` there is some special logic that adds both the normalized and unnormalized function names to the map, with the only difference being the value of `propto` for the call to `inline_function_statement` on the body.

So, I believe this impacts when a pdf is inlined into another pdf, do we treat it as if it was a updf or not. The changes are impacting stdlib function calls because the tests contain udfs which are defined in terms of stdlibs, which can't be themselves inlined further